src/pytorch_lightning/strategies/colossalai.py

```diff
@@ -48,6 +48,8 @@
     from colossalai.utils import get_current_device
     from colossalai.utils.model.colo_init_context import ColoInitContext
     from colossalai.zero import ZeroOptimizer
+else:
+    ColoInitContext = None
```

Comment:
```suggestion
    ColoInitContext = Any
```
Whenever you are defining a symbol to be used as a type, use `Any` instead of `None` so we don't run into errors such as https://github.com/Lightning-AI/lightning/pull/14809